iosdevapp/Model.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Model.swift
+//  iosdevapp
+//
+//  Created by Leonit Ejupi on 07.02.23.
+//
+
+import Foundation
+
+struct DataResponse: Decodable {
```

Comment:
Again it might be helpful to have some more descriptive naming e.g. `ImageURL` instead of `DataResponse`